CONTRIBUTING.md

```diff
@@ -79,13 +83,28 @@ ansible-test units --docker -v --python 3.8
 ansible-test units --docker -v --python 3.8 tests/unit/plugins/modules/net_tools/test_nmcli.py
 ```
 
+### Integration tests
+
 The following commands show how to run integration tests:
 
+#### In docker
+
+Integration tests on docker require two parameters:
```

Comment:
The product is called Docker (uppercase), not docker (lowercase). (That Docker's logo uses the word 'docker' in lower-case doesn't mean the product is called 'docker'.)
```suggestion
Integration tests on Docker require two parameters:
```
Also this statement itself is not correct, the test name isn't required.

Reply:
No, that looks like a bug in ansible-test, which we can also fix on our side by removing the invalid `cloud/gandi` entry in the aliases file of that test (I will do that in a minute).

Reply:
I'll rebase this PR on main and update the wording